src/containers/AdminDashboard/AdminDashboard.jsx

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react';
+import RenderCategories from './Category/RenderCategories';
+
+class AdminDashboard extends Component {
+	render() {
+		return (
+			<RenderCategories ></RenderCategories>
```

Comment:
```suggestion
			<RenderCategories />
```

det her burde fungere, hvis den ikke skal have nogle children